tests/unit/steps/test_generate_lz4.py

```diff
@@ -28,9 +28,11 @@ def setUp(self):
         )
 
     def test_get_command(self):
+        command = self.step.get_command(INPUT_FILE)
+        self.assertEqual('lz4', command.cmd)
         self.assertEqual(
-            self.step.get_command(INPUT_FILE).as_test_str(),
-            'lz4 -9 -f {}'.format(INPUT_FILE),
+            '-9 -f {}'.format(INPUT_FILE),
+            ' '.join(command.args)
```

Comment:
I suggest not using `.join` to test arrays, because these two args would return be equal, when in fact they aren't:
`['-9', '-f some_file.lz4']` # this command would fail, but it would pass your test
vs
`['-9', '-f', 'some_file.lz4']` # this is what you expect
You should assert arrays directly. You can use `assertSequenceEqual` for that:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertSequenceEqual